src/app/mainwindow.cpp

```diff
@@ -315,20 +315,19 @@ bool MainWindow::event(QEvent* event)
 
 void MainWindow::closeEvent(QCloseEvent* event)
 {
-    if (isVisible()) {
```

Comment:
looks silly indeed. it's been too long to remember the details, but i suspect an earlier version of the patch was trying to use QWidget::close() instead of QApp::quit().